applications/nrf5340_audio/src/bluetooth/le_audio.c

```diff
@@ -107,6 +107,9 @@ int le_audio_stop(enum audio_roles role)
 	int ret;
 
 	if (role == BROADCAST_SOURCE) {
+		if (!IS_ENABLED(CONFIG_TRANSPORT_BIS)) {
+			return -ENOTSUP;
+		}
```

Comment:
FYI (if someone are not aware):
This will actually remove broadcast_source_stop() in the linking stage, removing build warning/error